erpnext/hr/doctype/employee_advance/employee_advance.py

```diff
@@ -53,6 +53,10 @@ def set_total_advance_paid(self):
 		self.set_status()
 		frappe.db.set_value("Employee Advance", self.name , "status", self.status)
 
+	def get_due_advance_amount(self):
+		employee_due_amount = frappe.get_all("Employee Advance",filters={"employee":self.employee,"docstatus":1,"posting_date":("<=",self.posting_date)},fields=["advance_amount","paid_amount"])
```

Comment:
Break this line in multiple line, for better readability. Also, use space after comma 😉